server/src/test/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksServiceSpec.scala

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.dsp.workbench.welder.server
+
+import java.nio.file.Path
+import java.nio.file.Paths
+
+import cats.effect.IO
+import cats.effect.concurrent.Ref
+import io.chrisdavenport.log4cats.Logger
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import org.http4s.Uri
+import org.scalatest.{FlatSpec, Matchers}
+
+class StorageLinksServiceSpec extends FlatSpec with Matchers{
+
+  implicit val unsafeLogger: Logger[IO] = Slf4jLogger.getLogger[IO]
+
+  //TODO: remove boilerplate at the top of each test
+  "StorageLinksService" should "create a storage link" in {
+    val storageLinks = Ref.of[IO, Map[Path, StorageLink]](Map.empty)
+    val storageLinksService = StorageLinksService(storageLinks.unsafeRunSync())
+
+    val linkToAdd = StorageLink(Paths.get("/foo"), Uri.unsafeFromString("gs://foo/bar/baz.zip"), ".zip", true)
+
+    val addResult = storageLinksService.createStorageLink(linkToAdd).unsafeRunSync()
+    assert(addResult equals linkToAdd)
+  }
+
+  it should "not create duplicate storage links" in {
+    val storageLinks = Ref.of[IO, Map[Path, StorageLink]](Map.empty)
+    val storageLinksService = StorageLinksService(storageLinks.unsafeRunSync())
+
+    val linkToAdd = StorageLink(Paths.get("/foo"), Uri.unsafeFromString("gs://foo/bar/baz.zip"), ".zip", true)
+
+    storageLinksService.createStorageLink(linkToAdd).unsafeRunSync()
```

Comment:
nit, I'll use for comprehension and use `unsafeRunSync()` only once at the end of each test